selfdrive/locationd/paramsd.py

```diff
@@ -41,16 +44,34 @@ def handle_log(self, t, which, msg):
       yaw_rate = msg.angularVelocityCalibrated.value[2]
       yaw_rate_std = msg.angularVelocityCalibrated.std[2]
 
+      localizer_roll = msg.orientationNED.value[0]
+      roll_valid = msg.orientationNED.valid and ROLL_MIN < localizer_roll < ROLL_MAX
+      if roll_valid:
```

Comment:
Why isn't the std from the localizer used?

Why do you have to put in roll observation if invalid but not yaw?